initfiles/componentfiles/CMakeLists.txt

```diff
@@ -13,9 +13,10 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 ################################################################################
-
-ADD_SUBDIRECTORY(ftslave)
+if ( PLATFORM AND UNIX )
+   ADD_SUBDIRECTORY(ftslave)
+   ADD_SUBDIRECTORY(configschema)
+   ADD_SUBDIRECTORY(thor)
+   ADD_SUBDIRECTORY(launcher)
```

Comment:
Unless modification templates are necessary, only the configschema/xsd directory is really needed. Also, there will need to be a plugin directory added in the future for LN type builds. That work is not complete yet, however.
@xwang2713

Reply:
Replace "componentfiles/configxml" with "componentfiles/configschema/xsd". Builds are updated also.
@kenrowland back to you.